packages/compare-slider-block/src/settings.ts

```diff
@@ -0,0 +1,388 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    AssetChooserObjectType,
+    AssetInputSize,
+    DropdownSize,
+    IconEnum,
+    MultiInputLayout,
+    appendUnit,
+    defineSettings,
+    maximumNumericalOrPixelOrAutoRule,
+    minimumNumericRule,
+    numericalOrPixelRule,
+    presetCustomValue,
+} from '@frontify/guideline-blocks-settings';
+
+import { BorderStyle, getBorderRadiusSettings, getBorderSettings } from '../../shared';
+import { Alignment, Handle, Height, InheritSettings, LabelPlacement, heightMapWithPixel } from './types';
+import { FileExtensionSets } from '@frontify/app-bridge';
+
+export const settings = defineSettings({
+    main: [
+        {
+            id: 'alignment',
+            type: 'dropdown',
+            size: DropdownSize.Large,
+            defaultValue: Alignment.Horizontal,
+            choices: [
+                {
+                    value: Alignment.Horizontal,
+                    label: 'Left / Right',
+                },
+                {
+                    value: Alignment.Vertical,
+                    label: 'Top / Bottom',
+                },
+            ],
+        },
+    ],
+    basics: [
+        {
+            id: 'firstAssetSection',
+            type: 'sectionHeading',
+            label: 'First image',
+            blocks: [
+                {
+                    id: 'firstAsset',
```

Comment:
Use here a constant like in the other blocks

Reply:
done!